source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRChoiceSetViewDataSource.mm

```diff
@@ -247,4 +247,49 @@ - (NSString *)getTitlesOfChoices
     return [values componentsJoinedByString:@", "];
 }
 
+- (UIImage*) getRadioButtonForSelectedState{
```

Comment:
Please pull from origin master as logic of data source has changed. Images loading happens only in one place. A new UITableViewCell does the loading, and registered with the table for efficient recycling.